### DISCUSSION TITLE
EFB (Landing Calculation)

### DISCUSSION THREAD

**Discussion by Viper787** [magknight/787aviatorsTracker#573] [Q&A] [answered]
**Describe the bug**
EFB Landing calculation shows meters vs. feet for length, but shows the correct weight in lbs etc. If a pilot chooses LBS vs. KILOS for weight calculation chances are they are U.S. Based and would use feet vs. meters for all landing/takeoff calculations as well.

**To Reproduce**
Calculate a landing at any field.

**Expected behavior**
I should see runway measurements in feet vs. meters

**Screenshots**
attached

**Desktop (please complete the following information):**
 - OS: Mac 10.12.6
 - X-Plane version 11.50r3
 - Version 1.68 beta2
![Screen Shot 2021-04-10 at 4 26 36 PM](https://user-images.githubusercontent.com/56449761/114310963-a1ce5c00-9aa1-11eb-9395-b82dd4bf44b5.png)

**Additional context**

**Comment by aeroniemi** [comment, marked as answer]
You can select meters/feet in the EFB's settings:
![image](https://user-images.githubusercontent.com/13006900/114311842-4bd0d700-9ae8-11eb-8b31-8352b2b2585d.png)
![image](https://user-images.githubusercontent.com/13006900/114311846-512e2180-9ae8-11eb-9062-a9095de6f951.png)
